### ISSUE TITLE
AssignTargetFramework fails for PCL targeting .Net 4.6

### ISSUE THREAD

**Issue by jdudley84** [nuproj/nuproj#235]
Nuget package build fails for PCL targeting .Net 4.6. The problem can be worked around by switching the PCL to target .Net 4.5.

error MSB4018: The "AssignTargetFramework" task failed unexpectedly.
error MSB4018: System.ArgumentException: Value cannot be null or an empty string.
error MSB4018: Parameter name: profileValue
error MSB4018:    at NuGet.NetPortableProfile.Parse(String profileValue, Boolean treatOptionalFrameworksAsSupportedFrameworks)
at NuGet.VersionUtility.GetShortFrameworkName(FrameworkName frameworkName)
at NuProj.Tasks.Extensions.GetShortFrameworkName(FrameworkName frameworkName)
at NuProj.Tasks.AssignTargetFramework.ConvertToPackageFile(ITaskItem output)
at System.Linq.Enumerable.<>c__DisplayClass7_0`3.<CombineSelectors>b__0(TSource x)
at System.Linq.Enumerable.WhereSelectArrayIterator`2.MoveNext()
at System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()
at System.Linq.Buffer`1..ctor(IEnumerable`1 source)
at System.Linq.Enumerable.ToArray[TSource](IEnumerable`1 source)
at NuProj.Tasks.AssignTargetFramework.Execute()
at Microsoft.Build.BackEnd.TaskExecutionHost.Microsoft.Build.BackEnd.ITaskExecutionHost.Execute()
at Microsoft.Build.BackEnd.TaskBuilder.<ExecuteInstantiatedTask>d__26.MoveNext()

I think the issue is here: nuproj/src/NuProj.Task/Extensions.cs line 101

```C#
if (frameworkName.Identifier == ".NETPortable" && frameworkName.Version.Major == 5 && frameworkName.Version.Minor == 0)
{
	// Avoid calling GetShortFrameworkName because NuGet throws ArgumentException
	// in this case.
	return "dotnet";
}
```

It currently does a workaround for .NetPortable 4.5. Perhaps applying the same workaround to .NetPortable 4.6 would fix this issue.

**Comment by AArnott** [comment]
What does it mean for a PCL to target .NET 4.6? What other platforms are also targeted? What is the portable profile moniker that you expect to see in the built nupkg?

**Comment by jdudley84** [comment]
Creating a PCL via File->New->Project
![createpcl](https://cloud.githubusercontent.com/assets/9631047/15723974/d0010944-27f9-11e6-926b-dbb18822d570.PNG)

and create a nuget package project that references the PCL to produce a nuget package for it.

Select the targets for the PCL via Properties->Library->Targeting->Change

This configuration works:
![pcltargetsnet4 5](https://cloud.githubusercontent.com/assets/9631047/15723977/d4efc468-27f9-11e6-85a0-b05b6135bc03.PNG)

This configuration produces the above error when trying to build the nuget package project:
![pcltargetsnet4 6](https://cloud.githubusercontent.com/assets/9631047/15723979/d79ef792-27f9-11e6-818d-52455df1b273.png)

**Comment by kovalikp** [comment]
Ok, checked this, updating to NuGet.CommandLine 3.4.3 / NuGet.Core 2.11.1 should fix the issue.

**Comment by kovalikp** [comment]
Fixed by 7e01441e

**Comment by jdudley84** [comment]
Cool, thanks!
